docs/Help.md

```diff
@@ -38,7 +38,7 @@ If all else fails, you may be having connection issues to Twitch or your network
 
 <!--### Chatterino is failing to connect to chat-->
 
-### Chatterino is repeatedly dis/connecting from Twitch chat
+### Chatterino is repeatedly disconnecting from Twitch chat
```

Comment:
I think the `/` here is intended here to say that it keeps disconnecting or connecting, like the first sentence below:
`If you are having troubles with Chatterino repeatedly connecting/disconnecting, it is likely due to the large amount of channels you are connected to.`

Reply:
While this argument is fair, do you think that it makes sense for it to read as dis / connecting? I think disconnecting does a fine job as the title, as most people classify disconnecting+connecting as just disconnecting. [anyone I've seen complaining about this issue says 'my chatterino keeps disconnecting']

If anything it would be better if it was written as (dis)connecting in a similar way people write `x day(s) left until....` but I think the word itself just handles it well enough.

Reply:
The `/` was completely intentional, but is still incorrect. It should be `dis/reconnecting`. It reads as disconnecting OR reconnecting.

Reply:
> The `/` was completely intentional, but is still incorrect. It should be `dis/reconnecting`. It reads as disconnecting OR reconnecting, since your Chatterino could be disconnecting with no reconnection; or it could be reconnecting as a result of a dropped connection or packet loss, etc.

I don't think this looks very nice from a typographic point of view either. How about we just write `My Chatterino is repeatedly disconnevting and reconnecting to Twitch chat`? Or alternatively, just rephrase the whole sentence, e.g., `My Chatterino is stuck in a disconnect-reconnect loop` or similar.

Reply:
> Or alternatively, just rephrase the whole sentence, e.g., `My Chatterino is stuck in a disconnect-reconnect loop` or similar.

I like this take on it, because I was going to say, is there even a real reason why were are cutting down on just writing `disconnecting/reconnecting` besides just cutting down on a few characters? If we need the header to be specific both (although we currently only support information on one type of disconnection [though this probably will change in the future]) we just should just be obvious about it and prevent any confusion for the average user.

Reply:
I think the change as is is fair, disconnecting is one part the process but it's the 'troublesome' part, so it's fine to just highlight that imo.

Reply:
I'll go ahead and merge this in as is and we can PR/discuss this specific line in a separate PR